data/org_cluster_map_handler.py

```diff
@@ -0,0 +1,113 @@
+import json
+import logging
+import os
+import getorg
+
+from data.models import Contributor
+
+
+def handle(output_dir=None):
+    logger = logging.getLogger(__name__)
+
+    if not output_dir:
+        logger.debug('output_dir arg not provided! Setting output_dir'
+                     ' to cluster_map/')
+        output_dir = 'cluster_map'
```

Comment:
You can do `def handle(output_dir='cluster_map')` by default, if they provide some other dir it will be replaced by that.

You can also log that the `cluster_map` dir is used by default for this thing.